package.json

```diff
@@ -26,7 +26,8 @@
   "dependencies": {
     "buffer": "^6.0.3",
     "queue-microtask": "^1.1.2",
-    "readable-stream": "^4.0.0"
+    "readable-stream": "^4.0.0",
+    "relativize-url": "^0.1.0"
```

Comment:
Probably best to avoid dependency creep if possible - I assume that to write the relative URL functionality yourself isn't more than a few lines of code(?)